tests/ci/run_posix_sanitizers.sh

```diff
@@ -7,8 +7,20 @@ source tests/ci/common_posix_setup.sh
 build_type=Release
 cflags=("-DCMAKE_BUILD_TYPE=${build_type}")
 
-echo "Testing AWS-LC in ${build_type} mode with address sanitizer."
-build_and_test -DASAN=1 -DUSE_CUSTOM_LIBCXX=1 "${cflags[@]}"
+if [ $(dpkg --print-architecture) == "arm64" ]; then
+  # BoringSSL provides two sets tests: the C/C++ tests and the blackbox tests.
+  # Details: https://github.com/google/boringssl/blob/master/BUILDING.md
+  # The blackbox tests (run `go test` under `ssl/test/runner`) takes 30 minutes to complete on ARM when ASAN clang flag enabled.
+  # This does not happen on X86 ASAN and ARM (when ASAN disabled).
```

Comment:
NP bit unclear to me what is meant by this sentence.

Reply:
Which part is unclear?

> This does not happen on X86 ASAN and ARM (when ASAN disabled).

I mean the tests does not take 30 min on the test dimensions -- X86 ASAN and ARM (when ASAN disabled).
I can change it to
```
On X86 ASAN and ARM (when ASAN disabled), the blackbox tests only take less than 2 minutes to complete.
```